detection-rules/impersonation_blockchain.yml

```
@@ -6,9 +6,9 @@ source: |
   type.inbound
   and (
       sender.display_name =~ 'blockchain'
-      or iedit_distance(sender.display_name, 'blockchain') <= 1
-      or ilike(sender.email.domain.domain, '*blockchain.com*')
-      or any(body.links, iedit_distance(.href_url.domain.root_domain, 'blockchain') <= 1)
+      or strings.ilevenshtein(sender.display_name, 'blockchain') <= 1
+      or strings.ilike(sender.email.domain.domain, '*blockchain.com*')
+      or any(body.links, strings.ilevenshtein(.href_url.domain.root_domain, 'blockchain') <= 1)
```

unrelated to your PR but looks like a bug, looks like this should be

```suggestion
      or any(body.links, strings.ilevenshtein(.href_url.domain.root_domain, 'blockchain') == 1)
```